From f4358f17a62e7a42f72b02d8e8fbd6b760b69f01 Mon Sep 17 00:00:00 2001
From: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Date: Fri, 9 May 2025 00:36:52 +0000
Subject: [PATCH] ASoC: soc.h: remove snd_soc_disconnect_sync()
Git-commit: f4358f17a62e7a42f72b02d8e8fbd6b760b69f01
Patch-mainline: v6.16-rc1
References: jsc#PED-14430 jsc#PED-14297 jsc#PED-14024

There is no snd_soc_disconnect_sync() implementation, and no one is
using it. Let's remove it.

Signed-off-by: Kuninori Morimoto <kuninori.morimoto.gx@renesas.com>
Link: https://patch.msgid.link/87jz6qpql7.wl-kuninori.morimoto.gx@renesas.com
Signed-off-by: Mark Brown <broonie@kernel.org>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 include/sound/soc.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/include/sound/soc.h b/include/sound/soc.h
index e425394c3ef4..5d7b98f1dd71 100644
--- a/include/sound/soc.h
+++ b/include/sound/soc.h
@@ -469,8 +469,6 @@ static inline int snd_soc_new_compress(struct snd_soc_pcm_runtime *rtd)
 }
 #endif
 
-void snd_soc_disconnect_sync(struct device *dev);
-
 struct snd_soc_pcm_runtime *snd_soc_get_pcm_runtime(struct snd_soc_card *card,
 				struct snd_soc_dai_link *dai_link);
 
-- 
2.52.0

